saleor/graphql/shop/types.py

```diff
@@ -68,7 +68,7 @@ class Domain(graphene.ObjectType):
     url = graphene.String(description="Shop's absolute URL.", required=True)
```

Comment:
We could also change this value: "The absolute URL of the API".

Reply:
@maarcingebala done

Reply:
Looks good, but we need to also generate an updated schema.graphql with `python manage.py get_graphql_schema > saleor/graphql/schema.graphql`.